Babylon/groups/powerbi/groups/workspace/commands/create.py

```diff
@@ -0,0 +1,32 @@
+import logging
+
+from azure.core.credentials import AccessToken
+from click import Context
+from click import command
+from click import argument
+from click import pass_context
+
+from ......utils.decorators import output_to_file
+from ......utils.logging import table_repr
+from ......utils.typing import QueryType
+from ......utils.response import CommandResponse
+from ......utils.request import oauth_request
+
+logger = logging.getLogger("Babylon")
+
+
+@command()
+@pass_context
+@argument("workspace_name", type=QueryType())
+@output_to_file
+def create(ctx: Context, workspace_name: str) -> CommandResponse:
```

Comment:
This command can have --select option and set the created workspace key in deploy config